core/src/test/scala/com/raphtory/api/analysis/table/TestTableOutput.scala

```diff
@@ -1,37 +1,48 @@
 package com.raphtory.api.analysis.table
 
+import cats.effect.IO
+import cats.effect.Resource
 import com.raphtory.algorithms.generic.EdgeList
-import com.raphtory.Raphtory
 import com.raphtory.api.analysis.graphview.DeployedTemporalGraph
 import com.raphtory.api.time.Perspective
-import com.raphtory.internals.graph
+import com.raphtory.defaultConf
+import com.raphtory.internals.components.RaphtoryServiceBuilder
+import com.raphtory.internals.context.RaphtoryContext
+import com.raphtory.internals.graph._
 import munit.CatsEffectSuite
 
-import scala.collection.mutable
-
 class TestTableOutput extends CatsEffectSuite {
 
-  lazy val suiteGraph: Fixture[DeployedTemporalGraph] = ResourceSuiteLocalFixture(
-          "graph",
-          for {
-            g <- Raphtory.newIOGraph()
-            _  = g.addVertex(0, 0)
-            _  = g.addVertex(0, 1)
-            _  = g.addVertex(1, 2)
-            _  = g.addEdge(0, 0, 1)
-            _  = g.addEdge(1, 1, 2)
-          } yield g
-  )
-  override def munitFixtures                          = List(suiteGraph)
+  def run[R](f: DeployedTemporalGraph => R): R =
+    RaphtoryServiceBuilder
+      .standalone[IO](defaultConf)
+      .use { standalone =>
+        IO {
+          val ctx = new RaphtoryContext(Resource.eval(IO(standalone)), defaultConf)
+          ctx.runWithNewGraph() { graph =>
```

Comment:
can we have a runWithNewGraphIO version that doesn't call unsafeRunSync to be used inside IO contexts